index.html

```diff
@@ -25,65 +27,71 @@
-              src="./images/heroimg.jpg" 
+          <div class="column">
+            <h1 class="hero-title">Welcome to CodeStudio</h1>
+            <div class="hero-container">
+              <p class="hero-text">
+                We are here to help you learn how to code.
+              </p>
+              <p class="hero-text">Start a new career with Code Studio</p>
+              <a class="hero-btn" href="#">Signup</a>
+            </div>
+          </div>
+          <div class="column">
+            <img
+              class="hero-img"
+              src="./images/heroimg.jpg"
               alt="an open laptop with code displayed on a desk with a plant and a mug"
-            >
+            />
+          </div>
         </div>
       </section>
+
       <section class="section course-offer">
         <div class="container">
           <h2 class="course-offer__header">What we teach</h2>
           <ul class="offer">
             <li class="offer__item">
-              <img 
-                class="offer__img img-placeholder" 
-                src="./images/html.jpg" 
+              <img
+                class="offer__img img-placeholder"
+                src="./images/html.jpg"
                 alt="laptop's keyboard as a background with a word - html, in the center"
-              >
+              />
               <div class="offer__text-container">
                 <p class="offer__text">HTML</p>
               </div>
               <p class="offer__advert">Discover website structure with us.</p>
             </li>
             <li class="offer__item">
-              <img 
-                class="offer__img" 
-                src="./images/css.jpg" 
+              <img
+                class="offer__img"
+                src="./images/css.jpg"
                 alt="a fragment of CSS code displayed on the computer's screen"
-              >
+              />
               <div class="offer__text-container">
                 <p class="offer__text">CSS</p>
               </div>
               <p class="offer__advert">Learn how to make your website cool.</p>
             </li>
             <li class="offer__item">
-              <img 
-                class="offer__img" 
-                src="./images/javascript.jpg" 
+              <img
+                class="offer__img"
+                src="./images/javascript.jpg"
                 alt="a fragment of JavaScript code displayed on the computer's screen"
-              >
+              />
               <div class="offer__text-container">
                 <p class="offer__text">JavaScript</p>
               </div>
-              <p class="offer__advert">Give your website great functionality.</p>
+              <p class="offer__advert">
+                Give your website great functionality.
+              </p>
             </li>
             <li class="offer__item">
-              <img 
-                class="offer__img" 
-                src="./images/react.jpg" 
+              <img
```

Comment:
"img" tag can't have the slash "/" at the end - take it out (make sure you don't use prettier)